omnibus-base-goerli.toml

```diff
@@ -5,15 +5,18 @@ include = [
     "tomls/settings-testnet.toml",
     "tomls/core.toml",
     "tomls/permissions.toml",
-    #"tomls/collaterals/snx.toml",
     "tomls/collaterals/weth.toml",
     "tomls/pools/spartan-council.toml",
     "tomls/markets/spot/eth.toml",
-    "tomls/permit-all-createPool.toml",
-    "tomls/permit-all-registerMarket.toml",
     "tomls/permit-all-transferCrossChain.toml",
 ]
 
+[setting.target_preset]
+defaultValue = "main"
+
+[setting.salt]
+defaultValue = "staging"
```

Comment:
this should be the same as other testnets (ie empty, default value)

would remove so that cross chain support works later.